napari/layers/labels/_tests/test_labels.py

```diff
@@ -664,6 +664,18 @@ def test_contour_local_updates():
     )
 
 
+def test_data_setitem_multi_dim():
```

Comment:
Can you describe the bug you found in a docstring here? I still don't understand it after reading the code...

Reply:
added. Could you check?

Reply:
Yes, it's great!